Gem/Code/Source/Automation/ScriptManager.cpp

```diff
@@ -807,7 +807,7 @@ namespace AtomSampleViewer
         }
         ImGui::SameLine();
         ImGui::Text(
-            "%d/%d", AZStd::distance(m_wizardSettings.m_reportsOrderedByThresholdToInspect.begin(), m_wizardSettings.m_reportIterator) + 1,
+            "%ld/%lu", AZStd::distance(m_wizardSettings.m_reportsOrderedByThresholdToInspect.begin(), m_wizardSettings.m_reportIterator) + 1,
```

Comment:
I don't know what the correct conversion specifier to use for 'difference_type' is, and didn't see anything obvious at the top of a brief goole search, but if there is no clear correct option for all platforms, it might be worth casting the result of AZStd::distance to a size_t, int, etc. and then use the correct conversion specifier for that type in printf.

Reply:
Thanks, these should be fixed now.